package.json

```diff
@@ -202,9 +202,9 @@
     "@hapi/call": "^8.0.1",
     "@hapi/subtext": "^7.0.3",
     "@kubernetes/client-node": "^0.16.3",
+    "@ogre-tools/fp": "5.2.0",
```

Comment:
Offtopic, but I wonder why this is moving around all the time in PRs

Reply:
It happens all the time because when it first came into `master` someone added it manually to `package.json` instead of with `yarn add *` or `npm install -D` (this commands sorts packages by name before writing to `package.json`).
So whenever you add new package properly this one becomes naturally sorted.

Reply:
FYI @jansav

Reply:
Since now it's merged to `master` it should disappears from PR-s diff..